data/operators/shop/mall.json

```diff
@@ -14,6 +14,32 @@
         "operator:wikidata": "Q2287759",
         "shop": "mall"
       }
+    },
+    {
+      "displayName": "Unibail-Rodamco-Westfield",
```

Comment:
This conflicts with an existing `brand` entry for Westfield-branded malls in some of these countries:

https://github.com/osmlab/name-suggestion-index/blob/159234c5c34b3900753bd96367f48549160d09a3/data/brands/shop/mall.json#L93-L106

In iD, you can only choose a brand preset or an operator preset, not both, so when both details need to be tagged, there needs to be a single entry in NSI that represents the overlap.

Unfortunately, after 5ce3c226faf42507172e28f83e8206b6c02e4b53, this brand entry ended up conflating the operator with the brand. It should be:

```
brand=Westfield
brand:wikidata=Q106623253
operator=Unibail-Rodamco-Westfield
operator:wikidata=Q608518
```

This would be consistent with the other brand entry for Westfield-branded malls operated by Scentre Group.

Reply:
In next commit I moved the info about the operator Unibail-Rodamco-Westfield to the brand item, and the conflict during `npm run build` disappeared (I didn't see it the first time, sorry for the mistake)